Gemfile

```diff
@@ -28,6 +28,8 @@ group :development, :test do
   gem "pry-rails"
   gem "pry-nav"
   gem "faker"
+  gem "factory_girl_rails"
```

Comment:
I think this would cause a merge conflict with @andela-ooranagwa's branch. Ha added this line too.